tinker/tinker_controller.py

```diff
@@ -86,6 +87,29 @@ def decorated(*args, **kwargs):
 #         return False
 
 
+# *args because some of the menus are called with args and some with **kwargs
+def admin_permissions(self, route_base, *args):
+    # this handles everything but redirects and the program search admin menus because they require extra permissions
+    if route_base != '/admin/redirect' and route_base != '/admin/program-search':
```

Comment:
1) should these if blocks include elif's.
2) you do a - if not /admin/redirect. But the next if block does a - if == /admin/redirect. That seems duplicated
3) should these be direct matches? or should they use 'in' i.e: if route_base in '/admin/redirect'